css/style.css

```diff
@@ -231,3 +231,97 @@ footer h5 {
 .p3-text p {
   font-style: italic;
 }
+
+.main2 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
I don't think this line will help; you don't want the second section to wrap to the next line